dockers/docker-orchagent/supervisord.conf

```diff
@@ -71,6 +71,16 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=orchagent:running
 
+[program:kernel_arp_restore]
```

Comment:
what is the different between kernel_arp_restore and restore_neighbors below? it seems the similiar things.

Reply:
`restore_neighbors` is executed only during warmboot and restoration is by programming and sending arp requests. `kernel_arp_restore` is only programming to kernel (as dynamic entries) and done for config_reload scenario. Do you prefer we should combine the script?

Reply:
i recall for fast reboot we also restore the arp entries, which script is that? do we restart kernel arp entries for fast reboot?

Reply:
AFAIK, for fastboot, we only restore to APP_DB so that orchangent programs those ARP entry to ASIC and the [arp_update ](https://github.com/Azure/sonic-buildimage/blob/master/files/scripts/arp_update) script will send out requests to resolve arp.

Reply:
It looks like `restore_neighbors` is used primarily during warmboot. It looks like it does accomplish the same thing, but I am hesitant to change anything warmboot related in case it has other side effects.

Reply:
why fast_reboot cannot use the this kernel_arp_restore script? why only use this script in config reload scenario?

Reply:
it is fine not to combine with restore_neighbors, but why this arp_restore cannot be combined into swssconfig.sh and also use it for fast reboot?

Reply:
The issue with placing this in `swssconfig.sh` is that these kernel entries can only be programmed after the VLAN interface comes up, which happens after `swssconfig.sh` has exited

Reply:
it is ok not be combine. but can this be used for fast-reboot as well?